core/primitives-core/src/config.rs

```diff
@@ -30,4 +30,4 @@ impl AccountIdValidityRulesVersion {
 pub struct ViewConfig {
     /// If specified, defines max burnt gas per view method.
     pub max_gas_burnt: Gas,
-}
+}
```

Comment:
Please undo this change.

Reply:
fixed: https://github.com/near/nearcore/pull/9317/commits/817606c297c71f35117c254640228aca70334a35